libscanbuild/intercept.py

```diff
@@ -154,6 +154,19 @@ def intercept_compiler_wrapper(_, execution):
         logging.warning(message_prefix, 'io problem')
 
 
+def expand_cmd_with_response_files(cmd):
```

Comment:
Could you write pydoc comment for the method? A short one liner about that this method transform the command line arguments. Then a bit explanation, when it is happening (please mention windows, qmake with nmake as you did in the PR comment). And a short comments about the argument and return value. Please mention that this code might throw exception too.